website/docs/deployment.md

```diff
@@ -28,6 +28,8 @@ First, modify your `docusaurus.config.js` and add the required params:
 | `url` | URL for your GitHub Page's user/organization page. This is commonly https://_username_.github.io. |
 | `baseUrl` | Base URL for your project. For projects hosted on GitHub pages, it follows the format "/_projectName_/". For https://github.com/facebook/docusaurus, `baseUrl` is `/docusaurus/`. |
 
+In case you want to use your custom domain for GitHub Pages, just put your `CNAME` file in `static` folder.
```

Comment:
Can we drop the term “just”? I understand that the intention to say that it is straightforward. But for users who have never done it before it may not be as easy at all. I think it’s more friendly to keep a neutral tone in docs.